doc/tutorial.md

```diff
@@ -569,6 +569,114 @@ sp[3]
 See {doc}`/reference/algorithms/index` for details on graph algorithms
 supported.
 
+## Using NetworkX backends
+
+NetworkX can be configured to use separate thrid-party backends to improve
+performance and add functionality. Backends are optional, installed separately,
+and can be enabled either directly in the user's code or through environment
+variables.
+
+Several backends are available to accelerate NetworkX--often
+significantly--using GPUs, parallel processing, and other optimizations, while
+other backends add additional features such as graph database
+integration. Multiple backends can be used together to compose a NetworkX
+runtime environment optimized for a particular system or use case.
+
+Refer to the {doc}`/backends` section to see a list of currently available backends known to work with NetworkX.
+
+NetworkX uses backends by _dispatching_ function calls at runtime to
+corresponding functions provided by backends, either automatically via
+configuration variables, or explicitly by hard-coded arguments to functions.
+
+### Automatic dispatch
+
+Automatic dispatch is possibly the easiest and least intrusive means by which a
+user can use backends with NetworkX code. This technique is useful for users
+that want to write portable code that runs on systems without specific
+backends, or simply want to use backends with existing code that cannnot be
+modified.
+
+The example below configures NetworkX to automatically dispatch to a backend
+named `fast_backend` for all NetworkX functions that `fast_backend` supports.
+
+- If `fast_backend` does not support a NetworkX function used by the
+  application, the default NetworkX implementation will be used.
+
+- If `fast_backend` is not installed on the system running this code, an
+  exception will be raised.
+
+```{code-block}
+bash$> NETWORKX_BACKEND_PRIORITY=fast_backend python my_script.py
+```
+
+```{code-block}
+# file: my_script.py
+import networkx as nx
+G = nx.Graph()
+# populate the graph...
+pr = nx.pagerank(G)  # runs using backend from NETWORKX_BACKEND_PRIORITY, if set
+```
+
+The equivalent configuration can be applied to NetworkX directly to the code
+through the NetworkX `config` global parameters. This will override the
+corresponding environment variable, but results in slightly less code
+portability.
```

Comment:
```suggestion
corresponding environment variable, so backends can be changed during a script. But it results in slightly less portable code in that one line of code must be changed to switch the backend, instead of an environment variable change.
```

Reply:
Thanks, @dschult.  I made all the suggested changes, but I also had some updates queued which I didn't get pushed before you made these suggestions, so they don't look exactly the same.  Plus, I did some minor word smithing, so let me know what you think.